src/Symfony/Component/SignalRegistry/SignalRegistryInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\SignalRegistry;
+
+interface SignalRegistryInterface
```

Comment:
Do we really need an interface ? I don't think so.

Reply:
I think we do, for testability.

Reply:
We can test, even if it's not an interface...

Reply:
Sure, but less flexibly. 
Application does not care about pcntl, it only needs a contract.
Same applies to testing, esp. because  `SignalRegistry` is final and so can't be mocked, which is fine.

Reply:
OK, let's re-add it